### ISSUE TITLE
Change the bundler install statement in README and the cloud9 installation script to effectively avoid installation of the mysql2 gem

### ISSUE THREAD

**Issue by ryzokuken** [publiclab/plots2#957]
@jywarren So I had been having this problem since ever that `bundle install --without production` was not avoiding the mysql2 gem from being downloaded. This wouldn't be as big a problem in cloud9 as mysql is installed by default in that environment and it wasn't being used in development anyway, but in my local machine, it threw an error as the `libmysql` package wasn't installed. Usually, I'd just go ahead and install mysql, but now that we had decided to make stuff as foolproof as possible for newer contributors, I decided to look into it.

Our `Gemfile` contains:

```
...
# run with `bundle install --without production` or `bundle install --without mysql` to exclude this
group :mysql, :production do
  gem 'mysql2', '~> 0.3.20'
  # mysql 0.4.3+ causes a version mismatch, apparently, and demands 'activerecord-mysql2-adapter'
end

# ships with sqlite set up for easy setup during development
# run with `bundle install --without development` or `bundle install --without sqlite` to exclude this
group :sqlite, :development do
  gem 'sqlite3'
end
...
```

Firstly, we're making 2 groups having the exact same gems!? I mean, doesn't that just make it more confusing? The development/sqlite and production/mysql group pairs mean the exact same thing.

Also, I realized that `bundle install --without production` must've been installing `mysql2` anyway because it exists in another group (mysql).

**The Solution:**

Long story short, making it `bundle install --without mysql production`, however, skips both the groups, actually skipping the mysql installation. Works like a charm for me, we should either remove the redundant group naming or make the required changes in both README.md and the cloud9 installation script (won't really make a difference in the case of cloud9, but then such things might matter in the long run).

This could be made a first-timers-only issue no matter which course of action is chosen. 
What do you think?

**Comment by jywarren** [comment]
> bundle install --without production

I ALSO noticed this recently!!! Great problem solving. 

I think the original idea was to make **both** the distinction between production/dev and between the user's ability to just choose what DB they like, whether they're running development or not. But you're right, it adds confusion. 

I like your quick solution for now - `bundle install --without mysql production` and indeed this would make a good first-timers-only issue! Go for it, and thanks!

**Comment by 500swapnil** [comment]
I would like to work on this issue

**Comment by 500swapnil** [comment]
Will replacing this in Gemfile

"run with `bundle install --without production` or `bundle install --without mysql` to exclude this
group :mysql, :production do"

with

"run with 'bundle install --without mysql  production' to exclude this
group :mysql, :production do"

And the changes in README.environment work?

**Comment by ryzokuken** [comment]
@jywarren Wasn't this fixed?

**Comment by jywarren** [comment]
Yes, it was! I guess we didn't link from the PR. Thanks!
